pkg/controller/datavolume/import-controller.go

```diff
@@ -160,23 +184,50 @@ func (r *ImportReconciler) sync(log logr.Logger, req reconcile.Request) (dvSyncR
 }
 
 func (r *ImportReconciler) syncImport(log logr.Logger, req reconcile.Request) (dvSyncState, error) {
-	syncState, syncErr := r.syncCommon(log, req, nil, nil)
+	syncState, syncErr := r.syncCommon(log, req, r.cleanup, nil)
 	if syncErr != nil || syncState.result != nil {
 		return syncState, syncErr
 	}
-	if err := r.handlePvcCreation(log, &syncState, r.updateAnnotations); err != nil {
+
+	pvcModifier := r.updateAnnotations
+	if syncState.usePopulator {
```

Comment:
@alromeros  What about DV with VDDK source, I don't think we can use populator for that (currently)?  No checkpoint support

Reply:
The VDDK source can still be used but I'm not too sure if these annotations are relevant without multi-sage import.

Reply:
Yeah you are right, currently sources with multi-stage import (VDDK and ImageIO using checkpoint API) should continue using the old flow. But I guess we can use the populator with regular VDDK and ImageIO sources if they lack multi-stage annotations.

Reply:
so need to also add here a check if len(dataVolume.Spec.Checkpoints) > 0 ? is that a good enough check?

Reply:
I think that should be enough but let me review the API in case I'm missing something.